src/test/java/tests/RegisterTest.java

```diff
@@ -0,0 +1,32 @@
+package tests;
+
+import helper.DataHelper;
+import helper.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_object.RegisterPage;
+
+public class RegisterTest extends BaseTest {
+    RegisterPage registerPage = new RegisterPage();
+    String email = "";
+    String password = "";
+    String pid = "";
+
+    @Test(description = "User can create new account")
+    public void tc07_UserCanCreateNewAccount() {
+        LogHelper.info("Click register tab");
+        registerPage.clickRegisterTab();
+
+        LogHelper.info("Enter valid information into all fields and click Register button");
+        password = DataHelper.getRandomPassword(10);
```

Comment:
password can use the same without error, so no need to random it

Reply:
fixed, please check again